pkgs/development/tools/goperf/default.nix

```diff
@@ -20,10 +20,13 @@ buildGoModule rec {
 
   passthru.updateScript = writeShellScript "update-goperf" ''
     export UPDATE_NIX_ATTR_PATH=goperf
-    ${lib.escapeShellArgs (unstableGitUpdater { inherit (src) url; })}
+    ${lib.escapeShellArgs (unstableGitUpdater {
+      inherit (src) url;
+      hardcodeZeroVersion = true; # no releases
+    })}
     set -x
-    oldhash="$(nix-instantiate . --eval --strict -A "goperf.go-modules.drvAttrs.outputHash" | cut -d'"' -f2)"
-    newhash="$(nix-build -A goperf.go-modules --no-out-link 2>&1 | tail -n3 | grep 'got:' | cut -d: -f2- | xargs echo || true)"
+    oldhash="$(nix-instantiate . --eval --strict -A "goperf.goModules.drvAttrs.outputHash" | cut -d'"' -f2)"
+    newhash="$(nix-build -A goperf.goModules --no-out-link 2>&1 | tail -n3 | grep 'got:' | cut -d: -f2- | xargs echo || true)"
```

Comment:
Something must've changed about this at some point, because the custom updater code here was already broken before I touched anything:

```
error: attribute 'go-modules' in selection path 'goperf.go-modules.drvAttrs.outputHash' not found
       Did you mean goModules?
```

Otherwise an interesting approach that could be modified to handle weirder tag formats on a per-package basis.

Reply:
`goperf.goModules` is correct